web/src/app/components/home/home.component.scss

```diff
@@ -0,0 +1,102 @@
+@import "../../../styles/variables.scss";
+
+.homepage-container {
+  .mat-card {
+    box-shadow: unset !important;
```

Comment:
I haven't seen the use of `!important` in a while, I think there might be a better way, where does the `box-shadow` come from?